packages/plugin-list/test/Checklist.test.ts

```diff
@@ -2594,16 +2679,16 @@ describePlugin(List, testEditor => {
                                         '<ul class="checklist"><li class="checked">abc</li></ul><p>[]<br></p>',
                                 });
                             });
-                            it('should add an empty list item at the end of an indented list, then remove it', async () => {
+                            it('should add an empty list item at the end of an indented list, then outdent it', async () => {
```

Comment:
Here you changed the test because of `preserve`, right ? Yet you did not duplicate+skip like the others. Any reason ? Just wondering.